mlir/lib/Dialect/Rock/utility/builderUtils.cpp

```diff
@@ -21,6 +21,19 @@ using mlir::arith::ConstantOp;
 
 namespace mlir {
 namespace rock {
+FloatAttr getConstantScalarAttr(OpBuilder &b, Type elementType, float value) {
+  APFloat apValue(value);
+  auto constValue = b.getFloatAttr(elementType, apValue);
+  return constValue;
+}
+
+IntegerAttr getConstantScalarAttr(OpBuilder &b, Type elementType,
```

Comment:
Unused, not declared in headers?

Reply:
Thanks for noticing! I completely forgot to delete these functions.
Done